libs/langchain/langchain/llms/chatglm.py

```diff
@@ -105,12 +119,11 @@ def _call(
 
         try:
             parsed_response = response.json()
-
             # Check if response content does exists
             if isinstance(parsed_response, dict):
-                content_keys = "response"
+                content_keys = "data"
                 if content_keys in parsed_response:
-                    text = parsed_response[content_keys]
+                    text = eval(parsed_response[content_keys]["choices"][0]["content"])
```

Comment:
sorry i missed this, why do we need to eval here?

Reply:
Because the output of GLM will add `'` at both ends, the result is like `"'hello'"`.

Reply:
can we just do `response.strip(')` instead?

Reply:
I tried to use the `strip` function to remove only the first and last quotes, but I found that the output of GLM is still different from the output of Openai, the different is:
GLM output
```
hello, i'm llm\n how can i help you.
```
and Openai output
```
hello, i'm llm
 how can i help you.
```
, so I chose the `eval` function to ensure that subsequent parsing of the llm output will work properly